templates/config.php.j2

```diff
@@ -1,3 +1,20 @@
+{% macro insert_var(var) -%}{#- Formats a variable for inserting it into PHP -#}
+	{%- if var | type == "bool" -%}
+		{#- bool -#}
+		{{ var | bool | lower }}
+	{%- elif var | type == "list" -%}
+		{#- list -#}
+		[ {% for item in var %}{{ insert_var(item) }}, {# Note: trailing commas in lists are legal #}{% endfor %} ]
+	{%- elif var | type == "dict" -%}
+		{#- dict -#}
+		[ {% for k,v in var.items() %}'{{ k }}' => {{ insert_var(v) }}, {# Note: trailing commas in dicts are legal #}{% endfor %} ]
+	{%- elif var | type == "unicode" or var | type == "AnsibleUnsafeText" or var | type == "str" -%}
+		'{{ var }}'
+	{%- else -%}
+		{# number or unhandled case -> write verbatim -#}
+		{{ var }}
```

Comment:
Can we assume that this case is user error? Then we could just throw an error, for example by dividing by zero:

```suggestion
		{{ "Your engelsystem_config contains an invalid value"/0 }}
```

Reply:
I don't think this case is even possible, but yes, I would assume user error.